dev-docs/source/UserSupport.rst

```diff
@@ -36,3 +36,250 @@ Bugs and Error Reports
 6.	When any issue is completed naming a user, that user is contacted to let them know it will be available in the nightly build and next release.  The gatekeeper that merges the fix should message the appropriate developer, to remind them to contact the original reporter. This could simply be through adding a comment while merging that points this out.
 
 
+Troubleshooting
+###############
+
+This is a list designed to take a user through how to gain diagnostic information, particularly when Mantid (Workbench) fails to **launch**.
+
+For performance profiling check out our `recommended tools <http://developer.mantidproject.org/ToolsOverview.html#profiling>`_. 
+
+
+.. _Trouble_Windows:
+
+Windows
+=======
+
+1. Does the **splash screen** appear? Can get a rough idea how far through launch it stops.
+
+
+2. Does the **error reporter** appear? Is there a useful stacktrace? [If the errorreport won't send, the user can check with "Show Details"]
+
+
+3. Try launching from a command prompt:
+
+.. code-block:: python
+
+	C:\MantidInstall\bin\MantidWorkbench
+
+If this does not work, try launching with: 
+
+.. code-block:: python
+
+	cd C:\MantidInstall\bin
+	set QT_PLUGIN_PATH=%CD%\..\plugins\qt5
+	export PYTHONPATH=%CD%;%PYTHONPATH%
+	python -m workbench.app.main
+
+
+4. Does **Qt** import correctly? In a command prompt / terminal window, run the following:
+
+.. code-block:: python
+
+    C:\MantidInstall\bin\mantidpython.bat --classic
+    import qtpy.QtCore
+
+
+5. Do **Mantid Algorithms** import correctly?
+
+.. code-block:: python
+
+    C:\MantidInstall\bin\mantidpython.bat --classic
+    import mantid.simpleapi
+
+
+6. Turn off **Server Checks**: Open ``C:\MantidInstall\bin\Mantid.user.properties`` in any texteditor, add each code line to the end of the file and try to open Workbench after each.
+	a. Instrumentfile : ``UpdateInstrumentDefinitions.OnStartup = 0``
+	b. Mantidversion : ``CheckMantidVersion.OnStartup = 0``
+	c. Usage Reporting: ``usagereports.enabled = 0``
+	d. Project Recovery: ``projectRecovery.enabled=false``
+
+
+7. Try renaming **Config Files**:
+
+.. code-block:: python
+
+	cd %APPDATA%\mantidproject
+	mv mantidproject.ini mantidproject.ini.backup
+	# Try again to start workbench, if that doesn't work ...
+	
+	cd %APPDATA%\mantidproject
+	mv mantid mantidbackup
+	# Try again to start workbench
+
+Advanced options:
+
+8. Check the PATH for conflicts with Mantid:
+
+.. code-block:: python
+
+	echo %PATH%
+
+.. code-block:: python
+
+    cd C:\MantidInstall\bin\
+    python -c "import sys; import os; import pprint; pprint.pprint(sys.path); pprint.pprint(os.environ)"
+
+9. Check for conflicts with **numpy**: ``python -c "import numpy; print(numpy.__file__)"`` Anything outside of ``C:`MantidInstall`` could be a problem.
```

Comment:
Minor typo with the path *outside of ``C:\MantidInstall``* instread of *outside of ``C:`MantidInstall``*